packages/autocorrect/README.md

```diff
@@ -29,14 +29,12 @@ yarn add -D prettier prettier-plugin-autocorrect
 
 ## Usage
 
-Once installed, [Prettier plugins](https://prettier.io/docs/en/plugins.html) should be automatically recognized by Prettier. To use this plugin, confirm that it's installed and run Prettier using your preferred method. For example:
+Once installed, [Prettier plugins](https://prettier.io/docs/en/plugins.html) must be added to `.prettierrc`:
 
-```sh
-# npx
-npx prettier --write writing.md
-
-# yarn
-yarn prettier --write writing.md
+```json
+{
+  "plugins": ["prettier-plugin-autocorrect"]
+}
 ```
```

Comment:
Instead of removing this section, I'd still keep it in addition to the new clarification in the `.prettierrc`.

Reply:
Thanks, just added back!